pages/auth/setup.vue

```diff
@@ -192,8 +193,8 @@ export default {
       } else {
         this.password = '';
         this.$nextTick(() => {
-          this.$refs.password.focus();
-          this.$refs.password.select();
+          this.$refs.password.$refs.input.focus();
```

Comment:
I'd suggest adding focus and select methods to the password component. This is making auth/setup tightly coupled to implementation details.